Makefile

```diff
@@ -24,6 +24,10 @@ RUST_DIRS := $(foreach crate, $(CRATES), \
                $(patsubst devices/$(crate)%.yaml, \
                           $(crate)/src/$(crate)%/, \
                           $(wildcard devices/$(crate)*.yaml)))
+GENERICS := $(foreach crate, $(CRATES), \
+               $(patsubst devices/$(crate)%.yaml, \
+                          $(crate)/src/generic.rs, \
```

Comment:
This isn't right, we don't need to loop over all devices. It can just be:
```make
GENERICS := $(foreach crate, $(CRATES), $(crate)/src/generic.rs)
```
or even:
```make
clean-rs:
    rm -rf $(RUST_DIRS)
    rm -f */src/generic.rs
```

Reply:
Fixed. Rebased.